gaas-ui/src/rest/repositories/create-storetypes-graph-repo.ts

```diff
@@ -29,5 +34,5 @@ export interface ICreateGraphConfig {
     };
     proxyContextRoot?: string;
     proxyHost?: string;
-    proxyStores?: Array<{ graphId: string, url: string }>;
-}
\ No newline at end of file
+    proxyStores?: Array<{ graphId: string; url: string }>;
```

Comment:
Just wondering on the difference between semi-colon and comma here?

Reply:
Think its because of the object again